tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Install OpenVPN
+  import_tasks: install.yml
+  tags:
+    - install
+
+- name: Copy or Generate server keys
+  import_tasks: server_keys.yml
+  tags:
+    - server_keys
+
+- name: Enable ipv4 forwarding
+  sysctl:
```

Comment:
Not sure, but do you also need to write this to sysctl.conf so it's persistent between reboots?

Reply:
Also, you could use one task and `with_items`

Reply:
Actually no since the ansible module does this already "sysctl - Manage entries in sysctl.conf"

Reply:
a little bit ugly since ipv6 is only used when ipv6 is enabled